easybuild/easyconfigs/m/multicharge/multicharge-0.2.0-iimkl-2022a.eb

```diff
@@ -0,0 +1,41 @@
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeNinja'
+
+name = 'multicharge'
+version = '0.2.0'
+
+homepage = 'https://github.com/grimme-lab/multicharge'
+description = """Electronegativity equilibration model for atomic partial charges."""
+
+toolchain = {'name': 'iimkl', 'version': '2022a'}
+
+github_account = 'grimme-lab'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['953e2ace2f4035b1fa8ecf680f90b5ce6ad5caae17c8d8ccbc2578b92b69d3e7']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('Ninja', '1.10.2'),
+]
+
+dependencies = [
+    ('mctc-lib', '0.3.1'),
+    ('mstore', '0.2.0'),
+]
+
+configopts = '-DBLAS_LIBRARIES="-lmkl_sequential -lmkl_intel_lp64 -lmkl_core" '
+configopts += '-DLAPACK_LIBRARIES="-lmkl_sequential -lmkl_intel_lp64 -lmkl_core" '
```

Comment:
These should use $LIBBLAS and $LIBLAPACK respectively instead of manually listing the libs.
That way it will be correct even if one changes the toolchain to something else

Reply:
Thanks, that makes sense. I will try and do that in future.